apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx

```diff
@@ -24,167 +16,68 @@ export default class Results extends React.Component {
-            ))}
-          </tr>
-          </thead>
-          <tbody>
-          {
-            Object.entries(this.props.questions[session]['facilitator']).map(([question_key, question_data], i) => {
-              if (question_data['answer_type'] === 'selectValue') {
-                return (
-                  <tr key={i}>
-                    <td>
-                      {question_data['text']}
-                    </td>
-                    {
-                      this.state.facilitatorIds.map((id, j) => (
-                        <td key={j}>
-                          {he.decode(this.props.thisWorkshop[session]['facilitator'][question_key][id])}
-                        </td>
-                      ))
-                    }
-                  </tr>
-                );
-              }
-            })
-          }
-          </tbody>
-        </table>
-      );
-    } else {
-      return null;
-    }
-  }
-
-  renderFacilitatorSpecificFreeResponses(session) {
-    return (
-      <div>
-        {
-          Object.entries(this.props.questions[session]['facilitator']).map(([question_key, question_data], i) => {
-            if (question_data['answer_type'] === 'text') {
-              return (
-                <div key={i} className="well">
-                  {question_data['text']}
-                  {
-                    this.state.facilitatorIds.map((id) => {
-                      return this.renderFacilitatorSpecificBullets(
-                        this.props.thisWorkshop[session]['facilitator'][question_key],
-                        id
-                      );
-                    })
-                  }
-                </div>
-              );
-            }
-          })
-        }
-      </div>
-    );
-  }
+      if (!question || _.isEmpty(answers[questionId])) {
+        return null;
+      }
 
-  renderFacilitatorSpecificBullets(responses, facilitatorId) {
-    const hasResponses = responses && responses[facilitatorId];
-    return (
-      <li key={facilitatorId}>
-        {this.props.facilitators[facilitatorId]}
-        <ul>
-          {
-            hasResponses && responses[facilitatorId].map((response, i) => (
-              <li key={i} style={styles.facilitatorResponseList}>
-                {he.decode(response)}
-              </li>
-            ))
-          }
-        </ul>
-      </li>
-    );
+      if (question['answer_type'] === 'scale'|| question['answer_type'] === 'singleSelect') {
+        return (
+          <SingleChoiceResponses
+            question={question['text']}
+            answers={answers[questionId]}
+            possibleAnswers={question['options']}
+            key={i}
+            answerType={question['answer_type']}
+          />
+        );
+      } else if (question['answer_type'] === 'text') {
+        return (
+          <TextResponses
+            question={question['text']}
+            answers={answers[questionId]}
+            key={i}
+          />
+        );
+      }
+    }));
```

Comment:
What about [multi_select](https://github.com/code-dot-org/code-dot-org/pull/23371/files#diff-75f8f172f849ce785da2f56f73ec162fL34)?

Reply:
I left this out. We don't have any multi-select questions for the time being.